packages/fxa-content-server/tests/functional_settings.js

```diff
@@ -3,10 +3,10 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 module.exports = [
-  'tests/functional/settings/change_password.js',
-  'tests/functional/settings/connected_services_oauth_clients.js',
-  'tests/functional/settings/external_links.js',
-  'tests/functional/settings/recovery_key.js',
-  'tests/functional/settings/secondary_email.js',
-  'tests/functional/settings/two_step_auth.js',
+  //'tests/functional/settings/change_password.js',
```

Comment:
Do we need the `funcational_settings` test suite anymore?

Reply:
No we don't. But instead of deleting the whole suite I just commented out the tests.

Reply:
Cool, lets file a follow up to remove it.